libs/blocks/notification/notification.js

```diff
@@ -90,8 +90,15 @@ function decorateFlexible(el) {
     el.querySelector('.background'),
     el.querySelector('.foreground'),
     el.querySelector('.close'),
-  ];
+  ].reduce((acc, curr) => {
+    if (curr) return [...acc, curr];
+    return acc;
+  }, []).filter(Boolean);
```

Comment:
I'm still not clear on this 😅 Do you still need the reduce since you're using the filter?

Reply:
Oh man... I need more coffee. Fixing now 😅